README.rst

```diff
@@ -92,6 +92,30 @@ Build from git
    $ ./configure PKG_CONFIG_PATH=$PWD/../openssl/build/lib/pkgconfig:$PWD/../nghttp3/build/lib/pkgconfig LDFLAGS="-Wl,-rpath,$PWD/../openssl/build/lib"
    $ make -j$(nproc) check
 
+
+Build with BoringSSL without 0-RTT enabled
```

Comment:
Why "without 0-RTT enabled"?  ngtcp2 supports 0-RTT with boringssl.

Reply:
Yes, this section has nothing to do with 0-RTT whatsoever :)))  I fixed the description of this section as well as the section itself.    Some leftover from before I overlooked.  Thank you so much for reviewing it @tatsuhiro-t